### DISCUSSION TITLE
Content for Images page

### DISCUSSION THREAD

**Discussion by EmmaDawsonDev** [AccessibleForAll/AccessibleWebDev#84] [Ideas]
Let's discuss here the kind of content that should be included on the images page.

Thoughts so far:
Who to consider/who is affected?
Types of images - descriptive, functional, decorative
What is alternative text?
How to add alt text to descriptive, functional (include icons here or link to icons page) and decorative images(maybe separate sections with code snippets)

**Comment by ctoffanin** [comment]
This sounds pretty good to me. There is enough content to jot down for images.

**Comment by EmmaDawsonDev** [comment]
The content is being written here. Feel free to add comments:
https://docs.google.com/document/d/1adGswESUDAtaEcigW5UN0Zl9jiodFp2Eo7lMw7SJb0A/edit?usp=sharing

**Comment by krupalitrivedi** [comment]
The content looks good! I will go through it again and check for any edits and comment then
